sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsError.cs

```diff
@@ -4,21 +4,21 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// Text Analitics Error.
+    /// Text Analytics Error.
     /// </summary>
     public struct TextAnalyticsError
     {
         internal TextAnalyticsError(string code, string message, string target = null)
         {
-            Code = code;
+            ErrorCode = code;
```

Comment:
Why are we renaming this property `ErrorCode`?

Reply:
There's a precedent in the Azure SDK: https://github.com/Azure/azure-sdk-for-net/blob/3edb484a64afc93d6dc9be9d0947bbaad49f9763/sdk/core/Azure.Core/src/RequestFailedException.cs#L25